packages/google_maps_flutter/google_maps_flutter/android/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
 
     dependencies {
         implementation "androidx.annotation:annotation:1.1.0"
-        implementation 'com.google.android.gms:play-services-maps:17.0.0'
+        implementation 'com.google.android.gms:play-services-maps:18.0.2'
```

Comment:
Does this impact min SDK support?

Reply:
As per the docs: https://developers.google.com/maps/documentation/android-sdk/start the maps sdk requires min sdk 19  and the plugin has min sdk set to 20

> Set Minimum SDK to an SDK version compatible with your test device. You must select a version greater than the minimum version required by the Maps SDK for Android version 18.0.x, which is currently Android API Level 19 (Android 4.4, KitKat) or higher. See the [Release Notes](https://developers.google.com/maps/documentation/android-sdk/releases) for the latest information on the SDK version requirements

I also ran the tests locally and it passed successfully

Reply:
Looks like the build is failing, I upgraded Gradle to 3.5.4 and that seems to fix it. 
and modified the tests to fix the failure